google/cloud/aiplatform/metadata/experiment_resources.py

```diff
@@ -0,0 +1,717 @@
+            )
+
+    @staticmethod
+    def _is_tensorboard_experiment(context: context._Context) -> bool:
+        """Returns True is Experiment is a Tensorboard Experiment created by CustomJob."""
+        return constants.TENSORBOARD_CUSTOM_JOB_EXPERIMENT_FIELD in context.metadata
+
+    @property
+    def name(self) -> str:
+        """The name of this experiment."""
+        return self._metadata_context.name
+
+    @classmethod
+    def create(
+        cls,
+        experiment_name: str,
+        *,
+        description: Optional[str] = None,
+        project: Optional[str] = None,
+        location: Optional[str] = None,
+        credentials: Optional[auth_credentials.Credentials] = None,
+    ) -> "Experiment":
+        """Creates a new experiment in Vertex AI Experiments.
+
+        ```
+        my_experiment = aiplatform.Experiment.create('my-experiment', description='my description')
+        ```
+
+        Args:
+            experiment_name (str): Required. The name of this experiment.
+            description (str): Optional. Describes this experiment's purpose.
+            project (str):
+                Optional. Project where this experiment will be created. Overrides project set in
+                aiplatform.init.
+            location (str):
+                Optional. Location where this experiment will be created. Overrides location set in
+                aiplatform.init.
+            credentials (auth_credentials.Credentials):
+                Optional. Custom credentials used to create this experiment. Overrides
+                credentials set in aiplatform.init.
+        Returns:
+            The newly created experiment.
+        """
+
+        metadata_store._MetadataStore.ensure_default_metadata_store_exists(
+            project=project, location=location, credentials=credentials
+        )
+
+        with _SetLoggerLevel(resource):
+            experiment_context = context._Context._create(
+                resource_id=experiment_name,
+                display_name=experiment_name,
+                description=description,
+                schema_title=constants.SYSTEM_EXPERIMENT,
+                schema_version=metadata._get_experiment_schema_version(),
+                metadata=constants.EXPERIMENT_METADATA,
+                project=project,
+                location=location,
+                credentials=credentials,
+            )
+
+        self = cls.__new__()
+        self._metadata_context = experiment_context
+
+        return self
+
+    @classmethod
+    def get_or_create(
+        cls,
+        experiment_name: str,
+        *,
+        description: Optional[str] = None,
+        project: Optional[str] = None,
+        location: Optional[str] = None,
+        credentials: Optional[auth_credentials.Credentials] = None,
+    ) -> "Experiment":
+        """Get's experiment if one exists with this in Vertex AI Experiments. Otherwise creates this experiment.
```

Comment:
Nit: change first sentence to: "Gets experiment if one exists with this experiment_name in Vertex AI Experiments."